nixos/tests/nextcloud/with-declarative-redis-and-secrets.nix

```diff
@@ -122,5 +122,19 @@ in {
 
     # redis cache should not be empty
     nextcloud.fail('test "[]" = "$(redis-cli --json KEYS "*")"')
+
+    # Create a backup, nuke the instance, restore the backup & check that everything works
+    nextcloud.systemctl("start --wait nextcloud-update-db.service")
```

Comment:
Hmm I don't know why this isn't merged with the other tests :thinking: 

Gonna check later, but nothing we should do in this PR.